### PR TITLE
Rejuvenate cures all diseases

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Closes https://github.com/space-wizards/space-station-14/issues/7485

### CODE REVIEW

File: Content.Server/Disease/DiseaseSystem.cs [hunk 1]
```diff
@@ -186,6 +186,14 @@ private void CureDisease(DiseaseCarrierComponent carrier, DiseasePrototype disea
             _popupSystem.PopupEntity(Loc.GetString("disease-cured"), carrier.Owner, Filter.Entities(carrier.Owner));
         }
 
+        public void CureAllDiseases(DiseaseCarrierComponent carrier)
```
Comment (Zumorica) [inline Content.Server/Disease/DiseaseSystem.cs:189]: Should follow the [conventions for public APIs in Entity Systems](https://docs.spacestation14.io/en/getting-started/conventions#public-api-method-signature).

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Comment by Elijahrane** [comment]
one sec I forgot to stop bisecting

**Comment by Elijahrane** [comment]
okay this is fucked now

File: Content.Server/Administration/Commands/RejuvenateCommand.cs [hunk 2]
```diff
@@ -90,9 +90,9 @@ public static void PerformRejuvenate(EntityUid target)
                 entMan.RemoveComponent<JitteringComponent>(target);
             }
 
-            if (entMan.TryGetComponent<DiseaseCarrierComponent>(target, out var carrier))
+            if (entMan.HasComponent<DiseaseCarrierComponent>(target))
             {
-                EntitySystem.Get<DiseaseSystem>().CureAllDiseases(carrier);
+                EntitySystem.Get<DiseaseSystem>().CureAllDiseases(target);
```
Comment (Zumorica) [inline Content.Server/Administration/Commands/RejuvenateCommand.cs:95]: Honestly, just keep the `TryGetComponent` and pass the result to the method. That's what resolves were intended for.

**Comment by Elijahrane** [comment]
okay now I think

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Review by Zumorica — APPROVED** [review]

**Pull request merged**